pkg/client/k8s/client.go

```diff
@@ -18,6 +18,8 @@ import (
 	"context"
 	"path/filepath"
 
+	v12 "k8s.io/client-go/kubernetes/typed/core/v1"
+
```

Comment:
remove this blank line. Please import package to follow rule: https://github.com/alibaba/pouch/blob/master/docs/contributions/code_styles.md#rule003---import-packages